src/consensus/aft/raft_types.h

```diff
@@ -20,7 +21,7 @@ namespace aft
 {
   using Index = int64_t;
   using Term = int64_t;
-  using NodeId = uint64_t;
+  using NodeId = ccf::NodeId;
```

Comment:
Do we still need this alias, or should everything be using `ccf::NodeId` rather than `aft::NodeId`?